### ISSUE TITLE
RuntimeWarning: invalid value encountered in ulong_scalars

### ISSUE THREAD

**Issue by zhengjie6** [orsic/swiftnet#40]
When trying to run python eval.py configs/rn18_single_scale.py, i ran into the follow error in evaluate.py, line 17 and 30

> (swiftnet) fyp@fyp-P95-96-97Ex-Rx:~/Desktop/swiftnet$ python eval.py configs/rn18_single_scale.py
> Num images: 0
> Num images: 0
> Upsample layer: in = 128, skip = 64, out = 128
> /home/fyp/Desktop/swiftnet/models/util.py:44: UserWarning: Using conv type 1x1: <class 'torch.nn.modules.conv.Conv2d'>
>   warnings.warn(f'Using conv type {k}x{k}: {conv_class}')
> /home/fyp/Desktop/swiftnet/models/util.py:44: UserWarning: Using conv type 3x3: <class 'torch.nn.modules.conv.Conv2d'>
>   warnings.warn(f'Using conv type {k}x{k}: {conv_class}')
> /home/fyp/Desktop/swiftnet/models/util.py:62: UserWarning: 	Using skips: True (only skips: False)
>   warnings.warn(f'\tUsing skips: {self.use_skip} (only skips: {self.only_skip})', UserWarning)
> Upsample layer: in = 128, skip = 128, out = 128
> Upsample layer: in = 128, skip = 256, out = 128
> Batch size: 14
> Num params: 11,797,071 = 620,559(random init) + 11,176,512(fine tune)
> SPP params: 116,476
> 0it [00:00, ?it/s]
> 
> **/home/fyp/Desktop/swiftnet/evaluation/evaluate.py:17: RuntimeWarning: invalid value encountered in ulong_scalars**
>   **avg_pixel_acc = num_correct / total_size * 100.0**
**> Errors:
> /home/fyp/Desktop/swiftnet/evaluation/evaluate.py:30: RuntimeWarning: invalid value encountered in ulong_scalars
>   class_iou[i] = (TP / (TP + FP[i] + FN[i])) * 100.0**
> 	road IoU accuracy = nan %
> 	sidewalk IoU accuracy = nan %
> 	building IoU accuracy = nan %
> 	wall IoU accuracy = nan %
> 	fence IoU accuracy = nan %
> 	pole IoU accuracy = nan %
> 	traffic light IoU accuracy = nan %
> 	traffic sign IoU accuracy = nan %
> 	vegetation IoU accuracy = nan %
> 	terrain IoU accuracy = nan %
> 	sky IoU accuracy = nan %
> 	person IoU accuracy = nan %
> 	rider IoU accuracy = nan %
> 	car IoU accuracy = nan %
> 	truck IoU accuracy = nan %
> 	bus IoU accuracy = nan %
> 	train IoU accuracy = nan %
> 	motorcycle IoU accuracy = nan %
> 	bicycle IoU accuracy = nan %
> IoU mean class accuracy -> TP / (TP+FN+FP) = nan %
> mean class recall -> TP / (TP+FN) = 0.00 %
> mean class precision -> TP / (TP+FP) = 0.00 %
> pixel accuracy = nan %
> val: nan
> 0it [00:00, ?it/s]
> 
> Errors:
> 	road IoU accuracy = nan %
> 	sidewalk IoU accuracy = nan %
> 	building IoU accuracy = nan %
> 	wall IoU accuracy = nan %
> 	fence IoU accuracy = nan %
> 	pole IoU accuracy = nan %
> 	traffic light IoU accuracy = nan %
> 	traffic sign IoU accuracy = nan %
> 	vegetation IoU accuracy = nan %
> 	terrain IoU accuracy = nan %
> 	sky IoU accuracy = nan %
> 	person IoU accuracy = nan %
> 	rider IoU accuracy = nan %
> 	car IoU accuracy = nan %
> 	truck IoU accuracy = nan %
> 	bus IoU accuracy = nan %
> 	train IoU accuracy = nan %
> 	motorcycle IoU accuracy = nan %
> 	bicycle IoU accuracy = nan %
> IoU mean class accuracy -> TP / (TP+FN+FP) = nan %
> mean class recall -> TP / (TP+FN) = 0.00 %
> mean class precision -> TP / (TP+FP) = 0.00 %
> pixel accuracy = nan %
> train: nan
> 
> 
>
